scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -264,6 +275,21 @@ def load(self):
         else:
             self.apply_filters()
 
+        if self.scripting:
+            # Check if at least one provided script met the conditions.
+            # Summarize logs for all calls.
+            was_script_matched = False
+            for instance in self.instances.values():
+                was_script_matched = (
+                    was_script_matched
+                    or self.handle_additional_scripts(instance.platform.name, instance)
+                )
```

Comment:
it can be 
```suggestion
           was_script_matched = any(self.handle_additional_scripts(instance.platform.name, instance)  for instance in self.instances.values())
```